src/main/resources/static/js/thing.js

```diff
@@ -16,18 +17,18 @@ function getLocations() {
 	            var json = JSON.stringify(response, null, 4);
 	            var jsonparsed = JSON.parse(json);
 
-	            var list = $('#locations');
-	            list.empty().append(new Option('', '', null, null));
+	            var list = $("#locations");
+	            list.empty().append(new Option("", "", null, null));
 	            for (var i = 0; i < jsonparsed.length; i++) {
-	                var option = new Option(jsonparsed[i].name + ' (' + jsonparsed[i].frostId + ')', jsonparsed[i].name + ' (' + jsonparsed[i].frostId + ')', null, null);
-	                option.setAttribute('data-value', JSON.stringify(jsonparsed[i], null, 4));
+	                var option = new Option(jsonparsed[i].name + " (" + jsonparsed[i].frostId + ")", jsonparsed[i].name + " (" + jsonparsed[i].frostId + ")", null, null);
+	                option.setAttribute("data-value", JSON.stringify(jsonparsed[i], null, 4));
 	                list.append(option);
 	            }
 	            list.select2({
-	                placeholder: 'Choose a location',
-	                width: 'style',
+	                placeholder: "Choose a location",
+	                width: "style",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Mixed spaces and tabs. (no-mixed-spaces-and-tabs)](https://app.codacy.com/app/FraunhoferIOSB/ChillImport/pullRequest?prid=4696407)